src/Crypt32.Shared/Crypt32+KeySpec.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="KeySpec"/> nested type.
+    /// </content>
+    public partial class Crypt32
+    {
+        /// <summary>
+        /// An identifier that specifies the type of key.
+        /// </summary>
+        public enum KeySpec : uint
```

Comment:
There is already a `LegacyKeySpec` type defined in the NCrypt.Shared project that looks very similar to this one. Is it right that we redefine it here? Or should we share the definition?
Usually, we should mirror what happens in the native .h file: if it's defined twice there, we define it twice here. Otherwise, we should find a way to share it.